ui/apps/platform/src/Containers/Network/SidePanel/NetworkDeploymentOverlay/NetworkPoliciesDetail/NetworkPoliciesDetail.tsx

```diff
@@ -65,7 +65,7 @@ function NetworkPoliciesDetail({ policyIds }: NetworkPoliciesDetailProps): React
                             <Button
                                 className="download uppercase text-primary-600 p-2 text-center text-sm border border-solid bg-primary-200 border-primary-300 hover:bg-primary-100"
                                 onClick={downloadYamlFile(`${name}.yaml`, yaml, 'yaml')}
-                                tabIndex="-1"
+                                tabIndex={-1}
```

Comment:
nit: This is an obnoxious TS requirement, since we have to deviate from the literate suggestion in MDN.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
Agreed, especially since these all end up as strings anyway. It looks like this is React's TS interpretation of `HTMLElement` 🤷‍♂️